cli/config.go

```diff
@@ -0,0 +1,86 @@
+package cli
+
+import (
+	"encoding/json"
+	"errors"
+	"net/http"
+	"path"
+	"time"
+
+	"github.com/prometheus/alertmanager/cli/format"
+	"github.com/prometheus/alertmanager/config"
+	"github.com/spf13/cobra"
+	//flag "github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+// Config is the response type of alertmanager config endpoint
+// Duped in cli/format needs to be moved to common/model
+type Config struct {
+	Config      string            `json:"config"`
+	ConfigJSON  config.Config     `json:configJSON`
+	VersionINFO map[string]string `json:"versionInfo"`
```

Comment:
I think the go naming convention for non-acronyms would have this be `VersionInfo`